src/core/server/saved_objects/mappings/types.ts

```diff
@@ -45,7 +45,7 @@
  * @public
  */
 export interface SavedObjectsTypeMappingDefinition {
-  /** The dynamic property of the mapping. either `false` or 'strict'. Defaults to `false` */
+  /** The dynamic property of the mapping, either `false` or 'strict'. */
```

Comment:
Defaults to `strict` then?

Reply:
I removed "Defaults to..." because it doesn't actually add a default value anywhere in the code or in the mappings. But I think it's probably useful to say something like "If dynamic isn't specified the type mappings will inherit `dynamic: strict` from the top-level index mappings"